addOns/grpc/CHANGELOG.md

```diff
@@ -5,11 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-
-## [0.0.1] - 2024-05-21
+## Added
+- gRPC Variant Support Added
 
 ### Added
 - Features
   - ProtoBuf Message Decoding and Encoding in the message view panels.
 
-[0.0.1]: https://github.com/zaproxy/zap-extensions/releases/grpc-v0.0.1
```

Comment:
Looks like this got mangled while sorting out the merge/rebase.

It was released back on the 21st, the variant part should be under the Unreleased heading.